.evergreen/config.yml

```diff
@@ -860,19 +878,38 @@ axes:
           DEPENDENCIES: "lowest"
 
 buildvariants:
-# Test all PHP versions with latest-stable MongoDB and PHPC on Debian
+# Test all PHP versions with latest-stable MongoDB and PHPC, RHEL, and Ubuntu
 - matrix_name: "test-php-versions"
-  matrix_spec: { "os": "debian11", "mongodb-edge-versions": "latest-stable", "php-versions": "*", "driver-versions": "latest-stable" }
+  matrix_spec:
+    os:
+      - debian11
+      - debian10
+      - rhel90
+      # TODO: RHEL 8.x is not working yet
```

Comment:
What's the issue with RHEL 8.x? I noted that it was [omitted from the corresponding PHPC PR](https://github.com/mongodb/mongo-php-driver/pull/1442/files#diff-2bc841e86ce96b7b422ae203fd8315d0b2a461956cecbe0e096420656fc3fb12R1142). Do we need a ticket to track this?

Also, looking at that PR, I wonder if `rhel80` is even relevant if we're testing `rhel90` here. `rhel80` might be more relevant for PHPC.

`rhel82-arm64` may be relevant as an ARM64 platform (as an alternative to `ubuntu2204-arm64`), but perhaps not necessary for PHPLIB as much as it would be for PHPC.

Reply:
The toolchain was not installed on RHEL 8 hosts, which has now been done. PHPC will test on those, but I'm not convinced we need to test all PHP versions on those platforms.

Reply:
I concur that we don't need this for PHPLIB. Feel free to remove.